.github/workflows/tests.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install apt dependencies (Linux)
-      run: sudo apt-get install unrar
+      run: sudo apt-get install unrar libgl1-mesa-glx xvfb
       if: ${{ runner.os == 'Linux' }}
```

Comment:
```suggestion
      if: ${{ runner.os == 'Linux' }}
    - name: Setup headless display
      uses: pyvista/setup-headless-display-action@v1
```

Reply:
Let's actually put it before the "Install apt dependencies" so all actions are at the beginning and all custom steps are below.